cmd/installer/pkg/install/manifest.go

```diff
@@ -132,6 +132,14 @@ func NewManifest(label string, sequence runtime.Sequence, bootPartitionFound boo
 					Source:      fmt.Sprintf(constants.InitramfsAssetPath, opts.Arch),
 					Destination: filepath.Join(constants.BootMountPoint, label, constants.InitramfsAsset),
 				},
+				{
+					Source:      "/usr/install/amd64/systemd-boot.efi.signed",
+					Destination: "/boot/EFI/EFI/BOOT/BOOTX64.EFI",
```

Comment:
something temporary to be removed?

Reply:
I'm not sure if we should do this now or later, otherwise after installing booting off from this iso, it won't use UKI. Or we only care about iso generation here in this PR?

Reply:
but isn't this code common already, so it would affect any install of Talos?

my concern is that it doesn't use proper constants, it's hardcoded even if we don't use new secure boot/trusted boot.

Reply:
I understand that, but it needs to be done proper way - it should be optional, we should do a proper layout for the partitions, etc.

we need to detect and act differently depending on the way Talos got booted

Reply:
yeh, I'm not sure how yet, maybe we add a new kernel param and installer checks it?